### DISCUSSION TITLE
HomeKit: No picture/video - Home says camera is off

### DISCUSSION THREAD

**Discussion by superpower10** [koush/scrypted#894] [Q&A]
I have a eufy Outdoor Cam Pro. It's integrated with the RTSP Plugin into Scrypted.

In Scrypted the Stream works perfectly. I can see a snapshot and when I click on it the video stream runs perfectly.

In Apple Home it tells me the camera is "off" and shows a black screen.

Tried it with and without transcoding.

(Reolink Doorbell added with ONVIF works fine)

Settings in Scrypted:
<img width="553" alt="Bildschirmfoto 2023-06-16 um 10 16 31" src="https://github.com/koush/scrypted/assets/20533966/f23210d9-14b1-4352-a097-1505bf1d43a7">
<img width="542" alt="Bildschirmfoto 2023-06-16 um 10 16 12" src="https://github.com/koush/scrypted/assets/20533966/11014c51-928c-4941-a018-c1a24cbdaf07">
<img width="556" alt="Bildschirmfoto 2023-06-16 um 10 16 04" src="https://github.com/koush/scrypted/assets/20533966/e289ae1e-a680-4b56-b58d-cf099ebf597f">
<img width="542" alt="Bildschirmfoto 2023-06-16 um 10 15 52" src="https://github.com/koush/scrypted/assets/20533966/eaa5eb1a-c06b-4f6d-9e1c-ca675143fb4c">
<img width="536" alt="Bildschirmfoto 2023-06-16 um 10 15 46" src="https://github.com/koush/scrypted/assets/20533966/dbfbe5ee-d394-4b12-a1aa-182a7d53f22d">
<img width="541" alt="Bildschirmfoto 2023-06-16 um 10 15 35" src="https://github.com/koush/scrypted/assets/20533966/d4d9f13f-e2a8-40cb-aca5-fbf936de26f5">
<img width="576" alt="Bildschirmfoto 2023-06-16 um 10 20 37" src="https://github.com/koush/scrypted/assets/20533966/df5c1de0-0cc6-4ba1-983b-a95cce0cd0a1">

**Comment by tellok** [comment]
you forget to turn is recording options on in home app
